CameraBrowser/CameraBrowserSample/CameraBrowserSample.csproj

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
We should really create a .NET project instead of a legacy Xamarin.Mac project.

The good news is that I believe the project file will be very simple:

```xml
<?xml version="1.0" encoding="utf-8"?>
<Project Sdk="Microsoft.NET.Sdk">
	<PropertyGroup>
		<OutputType>Exe</OutputType>
		<TargetFramework>net7.0-macos</TargetFramework>
		<Nullable>enable</Nullable>
	</PropertyGroup>
</Project>
```

it would even be nice if we could try to multi-target to Mac Catalyst and see if that works:

```xml
<?xml version="1.0" encoding="utf-8"?>
<Project Sdk="Microsoft.NET.Sdk">
	<PropertyGroup>
		<OutputType>Exe</OutputType>
		<TargetFrameworks>net7.0-macos;net7.0-maccatalyst</TargetFrameworks>
		<Nullable>enable</Nullable>
	</PropertyGroup>
</Project>
```